rabbitmq/tests/test_openmetrics.py

```diff
@@ -142,13 +143,60 @@ def test_unaggregated_endpoint(endpoint, fixture_file, expected_metrics, aggrega
     _common_assertions(aggregator)
 
 
+@pytest.mark.parametrize(
+    'endpoint, fixture_file, expected_metrics',
+    [
+        pytest.param(
+            'detailed?family=queue_delivery_metrics',
+            "detailed-queue_delivery_metrics.txt",
+            {
+                'rabbitmq.queue.get.ack.count',
+                'rabbitmq.queue.get.count',
+                'rabbitmq.queue.messages.delivered.ack.count',
+                'rabbitmq.queue.messages.delivered.count',
+                'rabbitmq.queue.messages.redelivered.count',
+                'rabbitmq.queue.messages.acked.count',
+                'rabbitmq.queue.get.empty.count',
+            },
+            id="detailed, query queue_delivery_metrics family",
+        ),
+    ],
+)
+@pytest.mark.skipif(
+    RABBITMQ_VERSION < version.parse('4.0'),
+    reason=f"Skipping test because RABBITMQ_VERSION is {RABBITMQ_VERSION} (not greater than 4.0)",
+)
+def test_unaggregated_endpoint_v4(
+    endpoint, fixture_file, expected_metrics, aggregator, dd_run_check, mock_http_response
+):
+    mock_http_response(file_path=OM_RESPONSE_FIXTURES / fixture_file)
```

Comment:
```suggestion
    mock_http_response(file_path=os.path.join(OM_RESPONSE_FIXTURES, fixture_file))
```
To make sure this works on windows as well, let's use `os.path.join` here (make sure to add the necessary imports)